preload/syscalls.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * syscallgate -- syscall() GATE for the scratchbox2 preload library
+ *
+ * Copyright (C) 2023 Jolla Ltd.
+ * Contact: Franz-Josef Haider <franz.haider@jolla.com>
```

Comment:
I think we should skip adding contact addresses here.

The header misses the license, I would suggest: 
`` `
 *
 *  SPDX-FileCopyrightText: 2022, Jolla Ltd.
 *  SPDX-License-Identifier: GPL-2.0
 *
```

Reply:
added a header, is it ok like i did it?

Reply:
Yeah looks good, we should check the contact address